packages/patternfly-4/src/components/content/icons.js

```diff
@@ -58,6 +59,20 @@ class Icons extends React.Component {
           return <IconCard key={id} id={id} icon={Icon} name={name} />;
         })}
         </Gallery>
+        <EmptyState variant={EmptyStateVariant.full}>
+          { filteredIcons.length == 0 && 
+            <div>
+              <EmptyStateIcon icon={CubesIcon}/>
```

Comment:
Should this icon be relevant to there being no icons for the given search?

Reply:
Yeah @rachael-phillips could you point me towards an icon that you think fits this case? I think it has to be a React icon.

Reply:
@christiemolloy Could we use fa-search?